controllers/awsmachine_controller.go

```diff
@@ -836,6 +832,78 @@ func (r *AWSMachineReconciler) reconcileLBAttachment(machineScope *scope.Machine
 	return nil
 }
 
+func (r *AWSMachineReconciler) registerInstanceToV2LB(machineScope *scope.MachineScope, elbsvc services.ELBInterface, i *infrav1.Instance) error {
```

Comment:
Yeah, I had that at some point. I had something like declaring a deregister and register function value and setting it based on the LB type. I can do that again, but it was hurting readability, IMHO.

Reply:
One is calling `IsInstanceRegisteredWithAPIServerLB` and the other `IsInstanceRegisteredWithAPIServerELB`. 

I guess it's a bit hard to spot because it's a single-letter change. :D Should be more obvious.

But yeah, refactoring this made it unreadable with a largish switch for types.

And I'm hoping that classic LB will eventually be retired. :)